plugins/woocommerce/tests/php/src/Blocks/Patterns/PTKClientTest.php

```diff
@@ -147,6 +147,96 @@ function () {
 		remove_filter( 'pre_http_request', array( $this, 'return_request_failed' ) );
 	}
 
+	/**
+	 * Test fetch_patterns returns an WP_Error when the request is successful with one item with invalid payload.
+	 */
+	public function test_fetch_patterns_returns_an_wp_error_when_the_request_is_successful_with_invalid_payload() {
+		add_filter(
+			'pre_http_request',
+			function () {
+				return array(
+					'body'     => '[
+                        {
+                            "ID": 14870,
+                            "site_id": 174455321,
+                            "title": "Review: A quote with scattered images",
+                            "name": "review-a-quote-with-scattered-images",
+                            "html": "<!-- /wp:spacer -->",
+                            "categories": {
+                                "testimonials": {
+                                    "slug": null,
+                                    "title": null,
+                                    "description": null
+                                }
+                            }
+                        },
+						{
+                            "ID": 14870,
```

Comment:
Should the `ID` of the two patterns be a different number?

Reply:
It makes sense! I updated it with 66041700cd9fa296f6bd5b571b691f44d4a7be14